compose/docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: '3.8'
```

Comment:
Quedó afuera el campo de version

Reply:
Yes, docker compose deprecated the version field. It is correct to remove the version field